StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -151,6 +151,17 @@ extension PaymentSheet {
                 print("[Stripe SDK]: PaymentSheet could not offer these external payment methods: \(missingExternalPaymentMethods). See https://stripe.com/docs/payments/external-payment-methods#available-external-payment-methods")
             }
 
+            // Requirements for Instant Debits:
+            // - Is the link payment method enabled?
+            // - Is the us_bank_account payment method disabled?
+            // - Is your merchant in the enable_link_instant_debits gate?
+            // - Does link_funding_sources include BANK_ACCOUNT ?
+            // - Is it a deferred intent? (we currently don’t support ID for deferred payments)
+            // - (MAYBE) is FeatureFlags.instantDebits enabled?
+            if intent.supportsLink {
+                recommendedStripePaymentMethodTypes.append(.instantDebits)
+            }
```

Comment:
this needs updates

Reply:
Android:

- https://github.com/stripe/stripe-android/blob/e4af6224256322c6735e9abdf89ebf3fc96fb8a9/paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirement.kt#L61-L70

Reply:
```
if 
    intent.supportsLink, // Is the link payment method enabled?
    !intent.isDeferredIntent, // Is it a deferred intent? (we currently don’t support ID for deferred payments)
    !recommendedStripePaymentMethodTypes.contains(.USBankAccount), // Is the us_bank_account payment method disabled?
    intent.linkFundingSources?.contains(.bankAccount) == true // Does link_funding_sources include BANK_ACCOUNT ?
{
    recommendedStripePaymentMethodTypes.append(.instantDebits)
}
```